### DISCUSSION TITLE
move asdf-cmake here please

### DISCUSSION THREAD

**Discussion by amrox** [asdf-community/.github#26] [General]
There was an [existing cmake plugin](https://github.com/srivathsanmurali/asdf-cmake) but it hasn't been updated in awhile. I re-started with a recent plugin template, integrated the previous work from the older plugin and then added some enhancements. 

I would like to move the new plugin here please:

https://github.com/amrox/asdf-cmake

**Comment by smorimoto** [comment]
CC: @srivathsanmurali

**Comment by amrox** [comment]
bump @srivathsanmurali

I have tried contacting via email as well, no response yet.

**Reply by srivathsanmurali** [reply]
Sorry for not responding earlier. Currently, not really working with asdf-cmake, it would make sense for @amrox to maintain his plugin if it is being actively using and working on it.

**Reply by amrox** [reply]
thank you @srivathsanmurali! 🙏

**Comment by amrox** [comment]
@smorimoto we have permission from the previous plugin author

May we move forward with moving https://github.com/amrox/asdf-cmake here?

**Comment by amrox** [comment]
bump

Are any other maintainers willing to help with this?

**Reply by smorimoto** [reply]
Sorry for the late response. I was a little busy with my work.

1. I will invite you here
2. Move the plugin here
3. I will do the work necessary to import the repository
